ddsketch/store/store.go

```diff
@@ -5,14 +5,21 @@
 
 package store
 
+import (
+	"github.com/DataDog/sketches-go/ddsketch/pb/sketchpb"
+)
+
 type Store interface {
-	Add(index int)
+	Add(index int32)
```

Comment:
I will change this back to `int` since the `MaxIndexableValue()` and `MinIndexableValue()`of the mappings have been changed so that `index >= math.MinInt32` and `index <= math.MaxInt32`.